README.md

```diff
@@ -1 +1,49 @@
-# hello-microverse
\ No newline at end of file
+# Hello Microverse Project
+
+This project is been carried out for the purpose of gaining mastery of the following:-
+1.	Repository Creation
+2.	Branch Creation
+3.	Linters Configuration
+4.	Creating of commit action with descriptive commit messages.
+5.	Working Remotely
+
+## Built With
+
+•	Major languages: Html and CSS,
+•	Technologies used are: Git and GitHub
+
+## Getting Started*
+
+This is a way of setting up a project with different people from different parts of the working on the same project remotely.
+To get a local copy up and running follow these simple example steps.
+
+### Install
+
+Visual Studio
+Code 
+Git
+
+## Authors
+👤 Victor Osagie
+•	GitHub: [@efoosag](https://github.com/efoosag)
+•	Twitter: [@Victor Osagie](https://www.twitter.com/Victorosagie08)
+•	LinkedIn: [@Victor Osagie](https://www.linkedin.com /in/victor-osagie-a713ba22b/)
```

Comment:
[OPTIONAL] Please use `-` instead of `•` to make bullet points so that each point would be on a separate line.